api/router.go

```diff
@@ -75,6 +75,11 @@ func NewRouter(s *Server) *mux.Router {
 		"snapshotRevert": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotRevert),
 		"snapshotBackup": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotBackup),
 
+		"snapshotCRCreate": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotCRCreate),
```

Comment:
For these APIs, we don't need to forward the requests to the volume owner node.
For example:
```
"snapshotCRCreate": s.fwd.Handler(f(schemas, s.SnapshotCRCreate))
```

Reply:
Thank you. Fixed